### PR TITLE
15 create UI home page

### PR DESCRIPTION
**Pull request by jayha14**

### CODE REVIEW

File: components/card/cards-with-title/CardsWithTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { ReactNode } from 'react'
+import { CardBase } from '../card-base/CardBase'
+
+interface ICardsWithTitle {
+  title: string
+  children: ReactNode
+}
+
+export const CardsWithTitle = ({ title, children }: ICardsWithTitle) => {
```
Comment (ericchentch) [inline components/card/cards-with-title/CardsWithTitle.tsx:9]: find another solution for title of card

**Review by ericchentch — CHANGES_REQUESTED** [review]
Check comments

File: components/card/cards-with-title/CardsWithTitle.tsx (hunk 1 above)
Comment (jayha14) [reply components/card/cards-with-title/CardsWithTitle.tsx:9]: yes, sir!

**Review by ericchentch — APPROVED** [review]
LGMR

**Pull request merged**